### PR TITLE
TS-95-Refactor-FreeRooms-dialog-window

### LINKED ISSUE #95: Refactor: FreeRooms dialog window
###### _Acceptance Criteria:_
Restyle dialog window, change store state and create saga for this window

Estimate:  _1,5 days_

### PR DESCRIPTION
**Pull request by HiroFive**
This branch based on TS-88-refactor-Rooms-Page

### CODE REVIEW

File: frontend/src/actions/rooms.js [hunk 1]
```diff
@@ -1,47 +1,161 @@
...
+
+export const getFreeRoomsSuccess = (freeRooms) => ({
+    type: actionsType.GET_FREE_ROOMS_SUCCESS,
+    freeRooms,
+});
 
-export const clearRoomOne = () => ({
-    type: actionsType.CLEAR_ROOM_ONE,
+export const clearFreeRooms = () => ({
+    type: actionsType.CLEAT_FREE_ROOMS,
```
Comment (ViktoriiaKosovan) [inline frontend/src/actions/rooms.js]: change CLEAT to CREAR

File: frontend/src/styles/variables.scss [hunk 2]
```diff
@@ -1,10 +1,29 @@
-$btn-size: 20px;
 $schedule-button-size: 15px;
 
-$bold-border: 1px solid $thunck-color-hover;
+// $bold-border: 1px solid $thunck-color-hover;
```
Comment (ViktoriiaKosovan) [inline frontend/src/styles/variables.scss]: Do you need this?

**Review by ViktoriiaKosovan — APPROVED** [review]

File: frontend/src/styles/variables.scss (hunk 2 above)
Comment (HiroFive) [reply frontend/src/styles/variables.scss:3]: removed

Comment (HiroFive) [reply frontend/src/styles/variables.scss]: removed

**Review by Yashchemskyi-Oleksandr — APPROVED** [review]

File: frontend/src/actions/rooms.js (hunk 1 above)
Comment (HiroFive) [reply frontend/src/actions/rooms.js]: changed

**Pull request closed without merging**